src/mega-plusplus.coffee

```diff
@@ -43,6 +43,7 @@ module.exports = (robot) ->
   reasonsKeyword = process.env.HUBOT_PLUSPLUS_REASONS or 'raisins'
   maxPoints = process.env.HUBOT_PLUSPLUS_MAX_POINTS or 5
   reasonConjunctions = process.env.HUBOT_PLUSPLUS_CONJUNCTIONS or 'for|because|cause|cuz|as'
+  killjoyMessage = process.env.HUBOT_PLUSPLUS_KILLJOY_MESSAGE or 'Killjoy Mode™ has enforced a maximum change of ' + maxPoints  + ' points'
```

Comment:
I know the `™` for `Buzzkill Mode™` is _probably_ not an enforceable trademark.. but just to be safe, I went with `Killjoy Mod™` instead 🤷‍♂️ 

@regentgal lemme know whatcha think!